base/libgit2/consts.jl

```diff
@@ -271,12 +274,14 @@ Option flags for `GitRepo`.
                         CREDTYPE_USERNAME           = Cuint(1 << 5),
                         CREDTYPE_SSH_MEMORY         = Cuint(1 << 6))
 
+if LibGit2.version() >= v"0.24.0"
```

Comment:
Sorry, I've been busy lately,  and I miss this part with `USE_SYSTEM_LIBGIT2`. My system is Ubuntu 14.04 LTS and it has `libgit2` v0.19. It will be a very bad idea to use this ever.